parm/jcb-rdas/observations/atmosphere/adpsfc_airTemperature_181.yaml.j2

```diff
@@ -0,0 +1,320 @@
+     - obs space:
+         name: adpsfc_airTemperature_181
+         distribution:
+           name: "{{distribution}}"
+           halo size: 500e3
+         obsdatain:
+           engine:
+             type: H5File
+             obsfile: "data/obs/ioda_adpsfc.nc"
+             missing file action: "warn"
+           obsgrouping:
+             group variables: ["stationIdentification"]
+             sort variable: "pressure"
+             sort order: "descending"
+         obsdataout:
+           empty obs space action: "{{empty_obs_space_action}}"
+           engine:
+             type: H5File
+             obsfile: jdiag_adpsfc_airTemperature_181.nc
+             allow overwrite: true
+         io pool:
+           max pool size: 1
+         observed variables: [airTemperature]
+         simulated variables: [airTemperature]
+
+       obs operator:
+           name: VertInterp
+           vertical coordinate: air_pressure
+           observation vertical coordinate: pressure
+           observation vertical coordinate group: MetaData
+           interpolation method: log-linear
+           #gsi geovals:
+           #  filename: "obsout/sfc_tsen_geoval_2024052700.nc"
+           #  levels_are_top_down: False
+           variables:
+           - name: airTemperature
+
+       linear obs operator:
+         name: VertInterp
+
+       obs error:
+         covariance model: diagonal
+
+       obs localizations:
+         - localization method: Horizontal Gaspari-Cohn
+           lengthscale: 200e3 # orig
+
+       obs filters:
+         # ------------------
+         # airTemperature (181)
+         # ------------------
+         # Accept, reject, or passivate (monitor)
+         - filter: Perform Action
+           udescription: "iuse_check"
```

Comment:
I think the word `descriptor` in your PR's description is better than the word `udescription`.